tests/openvino/test_exporters_cli.py

```diff
@@ -134,6 +136,7 @@ class OVCLIExportTestCase(unittest.TestCase):
         "stable-diffusion-3": 6 if is_tokenizers_version("<", "0.20") or is_openvino_version(">=", "2024.5") else 2,
         "flux": 4 if is_tokenizers_version("<", "0.20") or is_openvino_version(">=", "2024.5") else 0,
         "flux-fill": 4 if is_tokenizers_version("<", "0.20") or is_openvino_version(">=", "2024.5") else 0,
+        "lfm2": 0, # TODO: looks like tokenizers fail to convert
```

Comment:
I didn't observe any issues with the direct tokenizer conversion and accuracy locally. Could you share the error logs?